### DISCUSSION TITLE
STT "recording" can't be enabled on Edge when using Browser STT

### DISCUSSION THREAD

**Discussion by leondape** [danny-avila/LibreChat#2968] [Troubleshooting]
### What happened?

When clicking the microphone for STT the icon turn red for a frame and the goes black again.
Recording cannot be initiated. 
This however is not the case when using external STT. Tested it with whisper.

### Steps to Reproduce

1. Click on mic

### What browsers are you seeing the problem on?

Microsoft Edge

### Relevant log output

```shell
Console gives no output
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
it works but Edge automatically blocks microphone access. I noticed I had to refresh for it to work with Edge, too.

![image](https://github.com/danny-avila/LibreChat/assets/110412045/db9615a0-8cac-4d3d-90dd-9a4f575ff8ef)

**Comment by leondape** [comment]
I can confirm that this is not the issue. Microphone is perfectly enabled and works with other apps like zoom or teams. 
It is only with librechat that it does not work. 
Works with chrome and safari. 
Just Edge does not enable it. The icon stays "black"

**Reply by danny-avila** [reply]
https://github.com/danny-avila/LibreChat/assets/110412045/95ddac76-b4f7-493b-b4c3-30e932c40213

This is on edge (using browser STT)

> navigator.userAgent
'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36 Edg/125.0.0.0'

![image](https://github.com/danny-avila/LibreChat/assets/110412045/e483c4ec-68e1-4d71-b01d-db08b6b68c5c)

**Reply by leondape** [reply]
okay I might have to check if there are any edge global blacklists that prevent it from working

**Reply by leondape** [reply]
I have checked. It works with flawlessly with external engines but does not with browser STT.
No Idea what could be causing it.
Unfortunately no console input and no Idea what I need to do to reproduce it.

**Reply by falkenbt** [reply]
I'm having the exact same issue. It works fine via chrome browser. UserAgent: 
`Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/136.0.0.0 Safari/537.36 Edg/136.0.0.0`
Any clue how to debug this further?

**Reply by fvd2** [reply]
> I'm having the exact same issue. It works fine via chrome browser. UserAgent: `Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/136.0.0.0 Safari/537.36 Edg/136.0.0.0` Any clue how to debug this further?

Same here, works fine in Chrome, but not in Edge (same exact UserAgent as yours). Did you manage to get any further @falkenbt ?

**Reply by falkenbt** [reply]
I believe this is a Microsoft policy setting when running edge managed by your corp, see https://learn.microsoft.com/en-us/deployedge/microsoft-edge-browser-policies/speechrecognitionenabled

**Comment by xoChrisCo** [comment]
Seems to be the case for Arc Browser as well. Works on Chrome on my Mac, but not Arc.

**Comment by parthpat12** [comment]
Browser STT doesn’t work reliably with iOS Safari or Brave.
